runtime/moonbeam/src/lib.rs

```diff
@@ -627,13 +636,15 @@ impl pallet_identity::Config for Runtime {
 	type MaxSuffixLength = MaxSuffixLength;
 	type MaxUsernameLength = MaxUsernameLength;
 	type WeightInfo = moonbeam_weights::pallet_identity::WeightInfo<Runtime>;
+	type UsernameDeposit = ();
+	type UsernameGracePeriod = ();
```

Comment:
Seem previous comments for moonbase runtime

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/eff30bd3ef8cecd3bbed397e753d39d07f58f77a